.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Remove large formatting change to avoid overwriting all python git history
```

Comment:
This is a placeholder - will add commit sha from  https://github.com/aws/deep-learning-containers/pull/2870 when it is merged